spec/services/claims/case_worker_claim_updater_spec.rb

```diff
@@ -113,7 +146,7 @@ module Claims
           expect(updater.claim.assessment.disbursements).to eq 0.0
         end
 
-        it 'if no state_reason are supplied' do
+        it 'if rejected and no state_reason are supplied' do
```

Comment:
could use a local shared_examples with a parameter to remove the duplication here:
e.g. for stuff i am doing
```
#near top of spec block
shared_examples "creates injection attempts" do
  it 'returns true' do
    is_expected.to be true
  end

  it 'creates an injection attempt' do
    expect{ run! }.to change(InjectionAttempt, :count).by(1)
  end
end
```

and use:

```
    context 'when injection succeeded' do
      let(:json) { valid_json_on_success }
      include_examples 'creates injection attempts'
    end

    context 'when injection failed' do
      let(:json) { valid_json_on_failure }
      include_examples 'creates injection attempts'
    end